### PR TITLE
Refactor project

### PR DESCRIPTION
**Pull request by neoziro**
- Refactor all project
- Remove Lerna
- Simplify code

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,12 +27,32 @@
...
+    "loader",
+    "reload"
+  ],
   "devDependencies": {
     "babel-cli": "^6.7.5",
     "babel-core": "^6.7.6",
     "babel-eslint": "^8.2.1",
     "babel-jest": "^22.1.0",
     "babel-plugin-dynamic-import-node": "^1.2.0",
+    "babel-plugin-external-helpers": "^6.22.0",
```
[the remark this answers is not in the document]
Comment (neoziro) [reply package.json]: Why?

File: test/hot.dev.test.js [hunk 2]
```diff
@@ -99,7 +99,7 @@ describe('hot (dev)', () => {
     }, 1)
   })
 
-  it('should trigger error in unmount in opened state', () => {
+  it.only('should trigger error in unmount in opened state', () => {
```
Comment (theKashey) [inline test/hot.dev.test.js]: My bad.

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (theKashey) [reply package.json]: It has a side effect on the final bundle, causing this bug - https://github.com/storybooks/storybook/issues/1320

So, I've just removed it, and double check that bundle is ok-ish.

File: src/proxy/inject.js [hunk 3]
```diff
@@ -148,21 +148,21 @@ function inject(target, currentGeneration, injectedMembers) {
             key,
             `(function REACT_HOT_LOADER_SANDBOX () {
           var _this = this; // common babel transpile
-          var _this2 = this; // common babel transpile          
+          var _this2 = this; // common babel transpile
```
Comment (theKashey) [inline src/proxy/inject.js]: This is ☠️, I know that _this3 exists.
In ideal we should or inject `regenerate` in a constructor, to make these variables visible, or detect the error on regenerate, and ask to disable arrow function transpiling.

**Pull request merged**